www/caddy/src/opnsense/service/templates/OPNsense/Caddy/rc.conf.d/caddy

```diff
@@ -3,6 +3,7 @@
 {% set generalSettings = helpers.getNodeByTag('Pischem.caddy.general') %}
 {% if generalSettings.enabled|default("0") == "1" %}
 caddy_enable="YES"
+caddy_admin="unix//var/run/caddy/caddy.sock|0220"
```

Comment:
for a service being able to drop privs and FreeBSD ports already overriding CADDY_ADMIN I think this should actually be the FreeBSD default.

Reply:
For reference https://github.com/opnsense/ports/blob/master/www/caddy/files/caddy.in#L68

Reply:
> for a service being able to drop privs and FreeBSD ports already overriding CADDY_ADMIN I think this should actually be the FreeBSD default.

Perhaps - but I suspect upstream FreeBSD doesn't see the same issue because they don't have `setup.sh` changing ownership beneath them, which would mean the default `0200` permissions that Caddy sets on creating the socket would be valid for the lifetime of the process.

Reply:
The fact it's relevant for us means caddy can be susceptible to the issue and FreeBSD port setting the socket over `localhost:2019` just means improving the situation in FreeBSD port sounds like a win-win.

After a bit of back and forth I just want to say your suggestion is very good and I don't see the harm in moving it to FreeBSD ports too.